engines/groovie/player.cpp

```diff
@@ -55,12 +57,22 @@ void VideoPlayer::setOverrideSpeed(bool isOverride) {
 	_overrideSpeed = isOverride;
 	if (_fps != 0) {
 		if (isOverride)
-			_millisBetweenFrames = 1000 / 26;
+			_millisBetweenFrames = 1000.0f / 26.0f;
```

Comment:
Ugh, is switching to floats necessary here?

Reply:
it makes the frame pacing way better and keeps the audio in sync, this also helps T7G

I guess it could be done with all integer math but it'd be more complicated code